tests/Routing/RoutingUrlGeneratorTest.php

```diff
@@ -128,7 +128,7 @@ public function testControllerRoutesWithADefaultNamespace()
         $route = new Illuminate\Routing\Route(['GET'], 'foo/bar', ['controller' => 'namespace\foo@bar']);
         $routes->add($route);
 
-        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => 'something\foo@bar']);
+        $route = new Illuminate\Routing\Route(['GET'], 'something/else', ['controller' => '\something\foo@bar']);
```

Comment:
I assume the `routes.php` is `app/Http/routes.php` it `RouteServiceProvider.php` still contain

```php
    public function map(Router $router)
    {
        $router->group(['namespace' => $this->namespace], function ($router) {
            require app_path('Http/routes.php');
        });
    }
```

Everything under `app/Http/routes.php` would append the root namespace because it is wrap in `$router->group()`.